docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.general.scss

```diff
@@ -36,10 +36,10 @@ $hb-animation-timing-function: ease-in-out;
 $hb-image-hover-scale: 1.07;
 
 // Menu Variables
-$hb-menu-light-border: 1px solid $su-color-driftwood;
-$hb-menu-link-background-hover: lighten($su-color-driftwood, 25%);
-$hb-menu-link-border-hover: $su-color-driftwood;
-$hb-menu-link-border-active: darken($su-color-driftwood, 45%);
+$hb-thin-border: 1px solid;
+$hb-menu-link-background-hover: 'gray-light';
```

Comment:
love the consolidation here!

Reply:
haha and then I flipped flop back to just wanting to know what these colors are in the code... IDK.

Reply:
Yeah, I'm on the fence. For the most part they already existed so if we decide they aren't as useful we can refactor them out in the future. They might make things more difficult when we add new themes to the system.